packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -94,7 +96,8 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     const combinationPromise = this.mediatorContextPreprocess.mediate({ context });
 
     // Parse query
-    let operation: Algebra.Operation = (await this.mediatorSparqlParse.mediate({ query })).operation;
+    let operation: Algebra.Operation = (await this.mediatorSparqlParse.mediate(
+      { query, queryFormat: context && context.queryFormat ? context.queryFormat : 'sparql', context })).operation;
```

Comment:
Would be interesting if the query format could be automatically detected. I.e., the test function of both sparql and graphql return whether it can parse (or both just try)